src/shorthands.jl

```diff
@@ -104,3 +104,215 @@ zlims!(lims::NTuple{2, Real}) = zlims!(current_scene(), lims)
+"""
+function xticks!(scene; xtickrange=xtickrange(scene), xticklabels=xticklabels(scene))
+    ticks!(scene, tickranges=(xtickrange, tickranges(scene)[2:end]...), ticklabels=(xticklabels, ticklabels(scene)[2:end]...))
+    return nothing
+end
+function xticks!(; xtickrange=xtickrange(scene), xticklabels=xticklabels(scene))
+    xticks!(current_scene(), xtickranges=xtickrange, xticklabels=xticklabels)
+end
+
+"""
+    yticks!([scene,]; ytickranges=ytickrange(scene), yticklabels=yticklabel(scene))
+
+Set the tick labels and range along all the y-axis. See also [`ticks!`](@ref).
+"""
+function yticks!(scene; ytickrange=ytickrange(scene), yticklabels=yticklabels(scene))
+    r = tickranges(scene)
+    l = ticklabels(scene)
+    if length(r) == 2
+        ticks!(scene, tickranges=(first(r), ytickrange), ticklabels=(first(l), yticklabels))
+    else  # length(r) == 3
+        ticks!(scene, tickranges=(first(r), ytickrange, last(r)), ticklabels =(first(l), yticklabels, last(l)))
+    end
+    return nothing
+end
+function yticks!(; ytickrange=ytickrange(scene), yticklabels=yticklabels(scene))
+    yticks!(current_scene(), ytickrange=ytickrange, yticklabels=yticklabels)
+end
+
+"""
+    zticks!([scene,]; ztickranges=ztickrange(scene), zticklabels=zticklabel(scene))
+
+Set the tick labels and range along all z-axis. See also [`ticks!`](@ref).
+"""
+function zticks!(scene; ztickrange=ztickrange(scene), zticklabels=zticklabels(scene))
+    ticks!(scene, tickranges=(tickranges(scene)[1:2]..., ztickrange), ticklabels=(ticklabels(scene)[1:2]..., zticklabels))
+    return nothing
+end
+function zticks!(; ztickrange=ztickrange(scene), zticklabels=zticklabels(scene))
+    zticks!(current_scene(), ztickrange=ztickrange, zticklabels=zticklabels)
+end
+
+###
+### Ticks rotations
+###
+"""
+    ticksrotations(scene)
+
+Returns the rotation of all tick labels.
+"""
+function ticksrotations(scene)
+    axis = scene[Axis]
+    @assert !isnothing(axis)
+    return axis.ticks.rotation[]
+end
+
+"""
+    xticksrotation(scene)
+
+Returns the rotation of tick labels along the x-axis. See also [`ticksrotations`](@ref)
+"""
+xticksrotation(scene) = first(ticksrotations(scene))
+
+"""
+    yticksrotation(scene)
+
+Returns the rotation of tick labels along the y-axis. See also [`ticksrotations`](@ref)
+"""
+yticksrotation(scene) = ticksrotations(scene)[2]
+
+"""
+    zticksrotation(scene)
+
+Returns the rotation of tick labels along the z-axis. See also [`ticksrotations`](@ref)
+"""
+zticksrotation(scene) = ticksrotations(scene)[3]
+
+"""
+    xticksrotation!([scene,] zangle)
+
+Set the rotation of all tick labels.
+"""
+function ticksrotations!(scene, angles)
+    axis = scene[Axis]
+    @assert !isnothing(axis)
+    scene[Axis].ticks.rotation[] = angles
+    return nothing
+end
+
+"""
+    xticksrotation!([scene,] xangle)
+
+Set the rotation of tick labels along the x-axis. See also [`ticksrotations!`](@ref).
+"""
+function xticksrotation!(scene, xangle)
```

Comment:
```suggestion
function xtickrotation!(scene=current_scene(), xangle)

Reply:
This one will only work if `xangle` becomes a keyword argument. Should I change it to a keyword arg.

Reply:
Ah no, nevermind then - makes sense.  We should probably restrict `scene::Scene` then, as well, for the purposes of dispatch.